### PR TITLE
Consul template upgrade

### PR DESCRIPTION
**Pull request by nhuray**
Hi guys,

This is a PR to upgrade Consul Template in our base image.

In this PR I upgraded Consul Template to version 0.19.4 (latest) in order to use the `indent` function in the project `docker-alertmanager`.

This Consul Template version has deprecated the function `key_or_default`, so I took care to replace this function by `KeyOrDefault` in all the Bandsintown projects.

Nicolas

PS: This PR must be merged before [docker-alertmanager#2](https://github.com/bandsintown/docker-alertmanager/pull/2)

### CODE REVIEW

**Review by msotoperdomo — APPROVED** [review]
👍  all seems good!

File: bin/alpine [hunk 1]
```diff
@@ -94,11 +94,8 @@ build(){
 
 build_image(){
   test -n "$VERSION" || { error "Please set --version argument to define the version you want to debug"; exit 1;}
-  info "Pulling the base image..."
-  run_command="$(cat ${BASE_DIR}/versions/${VERSION}/Dockerfile | grep FROM | sed 's/^FROM \(.*\)$/docker pull \1/')"
-  ${run_command}
   info "Building image '$IMAGE:$VERSION'..."
-  run_command="docker build -t $IMAGE:$VERSION -f $BASE_DIR/versions/$VERSION/Dockerfile ."
+  run_command="docker build --pull -t $IMAGE:$VERSION -f $BASE_DIR/versions/$VERSION/Dockerfile ."
```
Comment (bratislavml) [inline bin/alpine:98]: Here using `--pull` we always attempt to pull a newer version? Is it necessary?

**Review by bratislavml — APPROVED** [review]
👍

File: bin/alpine (hunk 1 above)
Comment (nhuray) [reply bin/alpine:98]: exactly !

**Pull request merged**